api/envoy/api/v2/core/protocol.proto

```diff
@@ -28,6 +28,14 @@ message HttpProtocolOptions {
   // If not specified, this defaults to 1 hour. To disable idle timeouts explicitly set this to 0.
   google.protobuf.Duration idle_timeout = 1;
 
+  // The maximum duration of a connection. The duration is defined as a period since a connection
+  // was established. If not set, there is no max duration. When max_connection_duration is reached
+  // the connection will be closed. If the connection is an HTTP/2 connection a drain sequence will
+  // occur prior to closing the connection. See :ref:`drain_timeout
+  // <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.drain_timeout>`.
+  // Note: not implemented for upstream connections.
+  google.protobuf.Duration max_connection_duration = 3;
```

Comment:
Please add a release note for this new feature with a ref link to the new configuration field.

Reply:
done.